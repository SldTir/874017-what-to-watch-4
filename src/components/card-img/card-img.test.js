import React from 'react';
import renderer from 'react-test-renderer';

import CardImg from './card-img.jsx';

const mock = {
  src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  name: `Fantastic Beasts`,
};

it(`CardImg is rendered correctly`, () => {
  const tree = renderer.create(<CardImg
    imagePath={mock.src}
    nameFilm={mock.name}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
