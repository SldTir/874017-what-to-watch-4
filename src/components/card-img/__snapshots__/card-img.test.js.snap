// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImg is rendered correctly 1`] = `
<img
  alt="Fantastic Beasts"
  height="175"
  src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  width="280"
/>
`;
